www/_posts/2018-09-26-cordova-cli-8.1.0.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+author:
+    name: Chris Brody
+    url: https://twitter.com/brodybits
+title:  "Cordova-CLI Release"
+categories: news
+tags: release tools
+---
+
+We are happy to announce that Cordova CLI `8.1.0` was released in September 2018.
+
+## Release Highlights
+
+The purpose of this release is to resolve `npm audit` issues, use recent releases of cordova-android and cordova-windows, and include some other recent updates.  Here are the important changes:
+
+* README.md fixes
+* [GH-333](https://github.com/apache/cordova-cli/pull/GH-333) Update to use `cordova-lib@8.1.0`
```

Comment:
These links don´t work in the blog post. You can open them, if you remove `GH-`.

Reply:
Fixed them in `master`.

Reply:
Build and deploy is running here: https://travis-ci.org/apache/cordova-docs/builds/433967727